src/figuresClasses.ts

```diff
@@ -1,11 +1,106 @@
-export interface Figure {}
+export interface Figure {
+  shape: `triangle` | `circle` | `rectangle`;
+  color: `red` | `green` | `blue`;
 
-export class Triangle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  readonly shape: 'triangle';
+
+  readonly color: 'red' | 'green' | 'blue';
+
+  a: number;
+
+  b: number;
+
```

Comment:
these blank lines may be unnecessary

Reply:
I totally agree with you Boss! Very nice observations, thank you very much. As for the spaces, unfortunately they were not done by me  they were made by mate's Lint script when testing my code. I'll get into those improvements right away.

Reply:
kkkkkkkkkkkk boa edu